script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -1185,7 +1204,8 @@ I said as I eat the curry rice, without looking at Arihiko's face.
 [sha:440d41bd8b7967f5fb6360fe99a97b39d7cc08f4]{
 -- Page 156, Offset 858.
 -- アレで文句を言ったらバチが当たるってもんか」
-If you complained about that, I would've hit you."
+// Reading as Shiki would suffer divine punishment if he were to complain about his time with the Arimas?
+Lighting would strike you down if you were to complain about the time you spent there."
```

Comment:
I think it is divine punishment yeah

'You'd be struck by lighting if you complained about the time you spent there'

Reply:
I prefer the stronger image from "strike you down"